README.md

```diff
@@ -107,6 +107,26 @@ provided by the devel module. Default values were used, except:
 As well as the database dump, the static files generated by the above process
 (user images, images embedded in content) have also been included.
 
+Drupal 8
+--------
+
+As above, aims to be realistic. Demo data is from the `devel_generate` module
```

Comment:
Would somebody familiar with devel_generate and drupal8 be able to easily upgrade the contents of this target in 6 months to a more recent version of drupal, etc?  I, for one, have no experience with devel_generate, so would like more detailed instructions on how each of the companion files (db dumps, static content, etc, etc) were generated.